dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WebSocketPublicStreamTestSuite.scala

```diff
@@ -26,6 +27,18 @@ class WebSocketPublicStreamTestSuite extends MatcherSuiteBase with HasWebSockets
     dex1.api.upsertRate(btc, 0.00011167)
   }
 
+  protected def squashOrderBooks(xs: TraversableOnce[WsOrderBook]): WsOrderBook = xs.foldLeft(WsOrderBook.empty) {
+    case (r, x) =>
+      WsOrderBook(
+        asks = r.asks ++ x.asks,
+        bids = r.bids ++ x.bids,
+        lastTrade = r.lastTrade.orElse(x.lastTrade),
+        timestamp = xs.toList.last.timestamp
+      )
+  }
+
+  protected def squashBalanceChanges(xs: Seq[Map[Asset, WsBalances]]): Map[Asset, WsBalances] = xs.foldLeft(Map.empty[Asset, WsBalances]) { _ ++ _ }
+
```

Comment:
This should be used in test for _private_ stream, not _public_, that is in WebSocket**Private**StreamTestSuite

Reply:
fixed